docs/reference/js/api.md

```diff
@@ -5,10 +5,20 @@ description: Developer resources for working in JavaScript with IPFS, the InterP
 
 # IPFS in JavaScript
```

Comment:
This page should discuss the various ways one can "interface with IPFS" with JavaScript.  Helia isn't the only way.  At the minimum, we should add the js-kubo-rpc-client.

Reply:
Added mention in https://github.com/ipfs/ipfs-docs/pull/1694/commits/b9150b343c6014dfe259a13f00e67e9878d868b9